src/Service.Tests/Unittests/SqlQueryExecutorUnitTests.cs

```diff
@@ -224,6 +224,12 @@ Mock<MsSqlQueryExecutor> queryExecutor
                 = new(provider, dbExceptionParser, queryExecutorLogger.Object, httpContextAccessor.Object);
 
             queryExecutor.Setup(x => x.ConnectionStringBuilders).Returns(new Dictionary<string, DbConnectionStringBuilder>());
+            queryExecutor.Setup(x => x.PrepareDbCommand(
+                It.IsAny<SqlConnection>(),
+                It.IsAny<string>(),
+                It.IsAny<IDictionary<string, DbConnectionParam>>(),
+                It.IsAny<HttpContext>(),
+                It.IsAny<string>())).CallBase();
```

Comment:
when you use `CallBase()` doesn't that not run the MsSqlQueryExecutor specific implementation?

Reply:
yes, it runs `MsSqlQueryExecutor` specific implementation